crates/storage/db/src/implementation/mdbx/tx.rs

```diff
@@ -232,11 +239,23 @@ impl<K: TransactionKind> MetricsHandler<K> {
             let open_duration = self.start.elapsed();
             if open_duration >= self.long_transaction_duration {
                 self.backtrace_recorded.store(true, Ordering::Relaxed);
+                let message = if cfg!(debug_assertions) {
```

Comment:
ah you need to use #[cfg here because cfg! still keeps the code just doesn't evaluate it